src/installer/tests/HostActivation.Tests/MultilevelSDKLookup.cs

```diff
@@ -91,7 +91,7 @@ public void Dispose()
         [Fact]
         public void SdkMultilevelLookup_Global_Json_Single_Digit_Patch_Rollup()
         {
-            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            if (!OperatingSystem.IsWindows())
```

Comment:
Many of these conditions should actually be attributes instead so that the tests are skipped and save time. Maybe you want to tackle this as part of this PR?

Reply:
@ViktorHofer done